src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $daysGap;
+  width: ($cellWidth * 7) + ($calendarPadding * 2) + ($daysGap * 6);
+  padding: $calendarPadding;
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    height: $cellHeight;
+    width: $cellWidth;
+    border: 1px solid #000;
+    background-color: #eee;
+    transition: transform 0.5s;
+
+    &:hover {
+      background-color: #ffbfcb;
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+
+  @for $day from 1 through 31 {
+    & :nth-child(#{$day})::before {
+      content: '#{$day}';
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+    }
+  }
```

Comment:
It's not necessary to set the font-family and font-size for each day separately. It would be better to set these properties once for the entire calendar. This would make your code more efficient and easier to maintain.